chromium/rules.js

```diff
@@ -1,10 +1,12 @@
+/* global enableMixedRulesets */
+/* global domainBlacklist */
+/* global exports */
+
 "use strict";
 // Stubs so this runs under nodejs. They get overwritten later by util.js
```

Comment:
If node isn't supported. Can we remove DBUG, INFO, etc entirely from this file?

Reply:
I don't think so.

Reply:
Since this file gets run under node, maybe it would be a good candidate for a file level `/*eslint-env node*/`.

Reply:
No, it would make ESLint ignore `/* exported */` directives.

Reply:
I'm confused, why do we need to stub *some* global vars like DBUG, INFO, and WARN. But not others like `enableMixedRulesets`, and `domainBlacklist`?

Reply:
cc @koops76

Reply:
Because only `rules.js` is required by rewriter.

Reply:
I see that. It looks like `rewriter.js` does not end up calling code that needs `domainBlacklist` and `enabledMixedRulesets`, so it doesn't break. Regardless, I don't think this PR would cause problems with `rewriter.js` since the removed `NOTE`, `VERB` etc are not needed in this file.